### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by VadimVelikanets**
Add linter and logger. Refactoring

### CODE REVIEW

File: src/middleware/auth.middleware.ts [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    const token = req.headers.authorization.split(' ')[1];
+    if (!token) {
+      return res.status(401).json({ message: req.headers });
+    }
+    const decoded = jwt.verify(token, process.env.SECRET_KEY);
+    req.user = decoded;
+    logger.info(decoded);
+    return next();
+  } catch (e) {
+    return res.status(401).json({ message: 'Auth message' });
```
Comment (MarynaBandarenka) [inline src/middleware/auth.middleware.ts]: I'd recomend to move Auth message to constant. Smth like UNAUTHORIZED="Some message in file with all messages"

File: src/models/Role.ts [hunk 2]
```diff
@@ -0,0 +1,11 @@
+import { Schema, model } from 'mongoose';
+
+const Role = new Schema({
+  value: {
+    type: String,
+    unique: true,
+    default: 'USER',
```
Comment (MarynaBandarenka) [inline src/models/Role.ts]: USER value should be imported from constant.

**Review by MarynaBandarenka — APPROVED** [review]

File: src/middleware/auth.middleware.ts (hunk 1 above)
Comment (VadimVelikanets) [reply src/middleware/auth.middleware.ts]: @MarynaBandarenka updated

File: src/models/Role.ts (hunk 2 above)
Comment (VadimVelikanets) [reply src/models/Role.ts]: @MarynaBandarenka updated

**Pull request merged**